pom.xml

```diff
@@ -35,12 +35,20 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.retry</groupId>
+			<artifactId>spring-retry</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-aspects</artifactId>
+		</dependency>
```

Comment:
Are these dependencies still required?

Reply:
`retry` dependency has been removed. `spring-aspects` is needed by `ConsumerSeekAware` interface.

Reply:
`ConsumerSeekAware` appears to come from `spring-kafka`?

Reply:
Fixed.